### DISCUSSION TITLE
Wie bekomme ich aus einem .pr5 eine .knxproj?

### DISCUSSION THREAD

**Discussion by jostec** [brutella/hkknx-public#176] [Q&A] [answered]
Hallo,
ich habe schon eine etwas ältere KNX-Installation, die noch auf der ETS 3.0f programmiert worden ist. Und dementsprechend habe ich nur eine .pr5 Projektdatei. Wie kann ich daraus eine .knxproj machen, um die in hkknx einlesen zu können? Oder gibt es einen Weg, wie ich meine Geräte von Hand in hkknx anlegen kann? Danke.

**Comment by brutella** [comment, marked as answer]
Wenn du die Gruppenadressen weisst, die für die verschieden Geräte (Lampen, Steckdosen, usw) verwendet werden, dann kannst du diese auch händisch eintragen. Eine Projektdatei wird nicht zwingend benötigt.

Wie du aus einer .pr5 Datei eine .knxproj Datei machst, weiss ich leider auch nicht.
